app/src/main/java/org/dhis2/usescases/searchTrackEntity/SearchRepositoryImpl.java

```diff
@@ -152,8 +151,7 @@
                 })
                 .toList().map(list ->
                         CollectionsKt.filter(list, item ->
-                                !(item instanceof PictureViewModel) &&
```

Comment:
We need to filter picture and coordinates fields so that they are not displayed in the attributes list in the TEI search card.

Reply:
check if this fits